### DISCUSSION TITLE
Custom button when viewing an order

### DISCUSSION THREAD

**Discussion by moritzlerch** [pretix/pretix#3279] [Q&A]
Hey, 

I'm currently trying to get my ticket shop to work properly. I would like to have a button on the order page. The button shouldn't be static over all of the orders but should rather contain some individual information per order to link to an external website that requires parameters. Can this somehow be achieved via a normal plugin or what would be the process to implement something like this?

If possible the button should contain a list of answers to the questions of a main ticket and some answers of add-on products in that order.

Thanks for your help :)

**Comment by raphaelm** [comment]
You could write a plugin that makes use of this signal:
https://docs.pretix.eu/en/latest/development/api/general.html#pretix.presale.signals.order_info_top

We also sell a plugin that can do most of this, it just can't reference the questions of add-on products:
https://marketplace.pretix.eu/products/digital/
https://docs.pretix.eu/en/latest/plugins/digital.html

**Reply by moritzlerch** [reply]
Hey, 

would it also be possible to connect this functionality to the seats plug-in? I was thinking about purchasing it, but it would only help me if there could be an extra button with the seat ids of an order provided as a list in the link.

Would I be able to get the seat-ids of an order with the custom plugin through a signal that already has been implemented in the core of pretix?

**Reply by raphaelm** [reply]
The plugin signal can also access seating data, yes.

The digital content plugin cannot deal with any kind of "lists", it would create a separate link per order position with the respective seat ID.
